### PR TITLE
fix: issues - 237、238

### PR DESCRIPTION
**Pull request by devchenyan**
fix issues:

- https://github.com/Magickbase/neuron-public-issues/issues/237
- https://github.com/Magickbase/neuron-public-issues/issues/238

image

### CODE REVIEW

File: packages/neuron-ui/src/components/Send/index.tsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ const Send = () => {
     validateTotalAmount(totalAmount, fee, balance)
   } catch (err) {
     if (isErrorWithI18n(err)) {
-      errorMessageUnderTotal = t(err.message)
```
Comment (yanguoyu) [inline packages/neuron-ui/src/components/Send/index.tsx:126]: > When the balance is insufficient, the copywriter can refer to the prompt information given by the UI design.

IMO, it's not a bug. 100 error code is different from 115.

**Review by WhiteMinds — APPROVED** [review]

**Comment by silySuper** [comment]
@devchenyan help to push this review.

**Review by yanguoyu — APPROVED** [review]

**Review by FrederLu — APPROVED** [review]

**Pull request merged**